package.json

```diff
@@ -57,6 +58,7 @@
   "dependencies": {
     "axios": "^0.17.1",
     "babel-polyfill": "^6.26.0",
+    "bootstrap": "^3.3.7",
```

Comment:
isnt this already pulled via patternfly?

Reply:
No, `patternfly-react` pulling `react-bootstrap` (which isn't pulling `bootstrap`).
We can't remove `bootstrap` but we can remove `react-bootstrap` :+1:

Reply:
My bad, `patternfly-react` -> `patternfly` -> `bootstrap`.

I should be able to remove `bootstrap` and `font-awsome` as well.